posixlib/src/main/resources/scala-native/signal.c

```diff
@@ -0,0 +1,89 @@
+#include <signal.h>
```

Comment:
Can you please add the following header to this file, allowing it to be ignored on Windows. I believe we would receive CI errors otherwise. 

```suggestion
#if defined(__unix__) || defined(__unix) || defined(unix) ||                   \
    (defined(__APPLE__) && defined(__MACH__))
#include <signal.h>
```

Reply:
Fixed.